attributes/default.rb

```diff
@@ -67,6 +67,11 @@
 default['hops']['root_url']                    = node['download_url']
 default['hops']['dist_url']                    = node['hops']['root_url'] + "/hops-" + node['hops']['version'] + ".tgz"
 
+default['hops']['fuse']['dist_url']            = node['hops']['root_url'] + "/hops-fuse-mount-" + node['hops']['version'] 
+default['hops']['fuse']['staging_folder']      = node['hops']['dir'] + "/hops-staging"
+default['hops']['fuse']['mount_point']         = node['hops']['dir'] + "/hopsfs-mount-point" 
```

Comment:
Are we only going to have 1 mount point for a hopsfs file system?
If so, why is it not /Projects ? Then you can mount your project as a subdirectory of the mount point on your containers.

Reply:
you can have multiple mount points. I have changed the default path to be /hopsfs similar to /dbfs in databricks